### ISSUE TITLE
Vertical align seems to fight Flexbox

### ISSUE THREAD

**Issue by roblevintennis** [AustinGil/bedrocss#13]
Heya @AustinGil :)

I suppose this is obvious when you see it, but I just debugged some code where I had previously centered an icon due to `align-items` but then having copied curated pieces of bedrocss into my own reset.css and/or opinions.css, the `vertical-align: middle` appears to _fight_ against Flexbox align-items and friend:

```css
...
svg {
  vertical-align: middle; /* removes weird space below images */
...
```

https://github.com/AustinGil/bedrocss/blob/master/src/media.css#L5

___

## Visual context

Here's the before / after behavior (note the checkmark's vertical centering against the input height):

![image](https://user-images.githubusercontent.com/142403/164948247-1fc3c3d9-7e49-4d01-9585-4085d35b69e9.png)

![Screen Shot 2022-04-23 at 5 37 16 PM](https://user-images.githubusercontent.com/142403/164948238-08f6567c-61a5-4853-975a-a3ea45b79db8.png)

I suppose the question would be is it more or less likely that I'm going to be applying flexbox alignment to one of these media 🤔 e.g. do I just make the single override e.g. `unset` or `initial`, or, should this not be in the reset ❓ I'm not sure I have an answer -- any thoughts from your end?

**Comment by AustinGil** [comment]
I see. So you think it's better to remove SVG from that rule?

**Comment by roblevintennis** [comment]
Yeah, it would seem reasonable, but I'm wondering if maybe others like `img` might often use flexbox centering too and I honestly don't know for certain. This is the type of thing I do like once every few months so I don't recall 😋 I suppose you could ask the question which of these is likely to NOT get flexbox aligned?

In this case, since my issue is deep within an `Icon` component code that's within another `Input` component, it was hard to just do `unset` so I opted to just removed the vertical-align middle from my demo app's globals (which I've hand moved over these sorts of things). Luckily, I've opted to copy in this "opinion" into my global.css (it's an Astro app) -- so I just changed it to this:

```
:where(img, picture, svg, canvas, video, iframe) {
  max-inline-size: 100%;
  block-size: auto;
}
```

**Comment by AustinGil** [comment]
Yeah, I'd leave it on img for sure, but easy enough to remove from SVG. SVG is really just there for the other rules, but img for sure should have that vertical align.

**Comment by roblevintennis** [comment]
Ok, I'll go back and put it in and just remove SVG -- thanks! Feel free to leave open or close :)

**Comment by AustinGil** [comment]
Yeah, I defo think this one does more harm than good in it's current state. Thanks for reporting
